app/workers/asset_manager_delete_asset_worker.rb

```diff
@@ -0,0 +1,9 @@
+class AssetManagerDeleteAssetWorker < WorkerBase
+  def perform(legacy_url_path)
+    gds_api_response = Services.asset_manager.whitehall_asset(legacy_url_path)
+    asset_url = gds_api_response['id']
+    asset_id = asset_url[/\/assets\/(.*)/, 1]
```

Comment:
Not for this PR - but it might be nice to add this parsing of the URL ID to the API adapter gem at some point. Perhaps open an issue against that library?